autorest/multiapi/models/client.py

```diff
@@ -43,9 +44,28 @@ def __init__(
 
     def imports(self, async_mode: bool) -> FileImport:
         imports_to_load = "async_imports" if async_mode else "sync_imports"
-        return FileImport(
+        file_import = FileImport(
             json.loads(self.default_version_metadata["client"][imports_to_load])
         )
+        if TypingSection.REGULAR in file_import.imports:
```

Comment:
this code seems really complicated, what is it trying to do?

Reply:
i feel like there's going to be a better way of doing what this code is trying to do